tests/backend/test_webhook_handler.py

```
@@ -0,0 +1,101 @@
+import pytest
+from unittest.mock import patch, Mock
+from src.backend.webhook_handler import app, WebhookHandler, set_webhook_handler
+from src.backend.exceptions import WebhookError
```

Ruff (0.8.2)

`src.backend.exceptions.WebhookError` imported but unused

Remove unused import: `src.backend.exceptions.WebhookError`

(F401)